README.md

```diff
@@ -42,18 +42,22 @@ confluent-kafka-dotnet is distributed via NuGet. We provide three packages:
 To install Confluent.Kafka from within Visual Studio, search for Confluent.Kafka in the NuGet Package Manager UI, or run the following command in the Package Manager Console:
 
 ```
-Install-Package Confluent.Kafka -Version 0.11.4-RC2
+Install-Package Confluent.Kafka -Version 0.11.4-RC3
 ```
 
 To add a reference to a dotnet core project, execute the following at the command line:
 
 ```
-dotnet add package -v 0.11.4-RC2 Confluent.Kafka
+dotnet add package -v 0.11.4-RC3 Confluent.Kafka
 ```
 
-Pre-release nuget packages are available from the following nuget package source (Note: this is not a web url - you should specify it in the nuget package manger):
-[https://ci.appveyor.com/nuget/confluent-kafka-dotnet](https://ci.appveyor.com/nuget/confluent-kafka-dotnet). The version suffix of these nuget packages matches the appveyor build number. You can see which commit a 
-particular build number corresponds to by looking at the 
+We have started working towards a 1.0 release of the library which will occur after we add idempotence and transaction features. In order to best accomodate these and other changes,
```

Comment:
With this section getting bigger maybe add a section header, something like "Development branch"

Reply:
good idea